### DISCUSSION TITLE
Content rating update job has updated the rating in neo4j as well as redis, elastic search as individual content level but the same was not updated in cassandra content hierarchy

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Lern/Community#177] [Issues]
Hi Team,

Content rating update job has updated the rating in neo4j as well as redis, elastic search as individual content level but the same not updated in Cassandra content hierarchy.
Can you please help let us there are any job that will synch the data neo4j to Cassandra. 
can you share any document or KT video where we can understand the complete workflow of content rating update in database and how to come on the UI level.

**Comment by Jayaprakash8887** [comment]
@kamal-singhal Kindly raise this query in ED/KNOWLG BB. LERN BB Team will not be able to answer this.

**Reply by kamal-singhal** [reply]
@Jayaprakash8887 Thanks for this info. Please close it, i have raised issue in ED & Knowledge BB.
